### PR TITLE
Support Symfony3

### PR DESCRIPTION
**Pull request by tabbi89**

### CODE REVIEW

**Comment by pkruithof** [comment]
Thanks for the patch! Could you add the version to the travis config file as well?

**Comment by tabbi89** [comment]
No problem I will update this

**Comment by tabbi89** [comment]
@pkruithof Everything seems fine.

File: src/TreeHouse/WorkerBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -13,58 +13,58 @@ services:
     class: %tree_house.worker.queue_manager.class%
     arguments:
       - ~ # injected in extension
-      - @event_dispatcher
-      - @logger
+      - '@event_dispatcher'
```
Comment (pkruithof) [inline src/TreeHouse/WorkerBundle/Resources/config/services.yml:16]: Are these quotes necessary for anything?

Comment (tabbi89) [inline src/TreeHouse/WorkerBundle/Resources/config/services.yml:16]: Scalars need to be quoted in the configuration for Symfony 3

**Comment by pkruithof** [comment]
Thanks!

**Pull request merged**